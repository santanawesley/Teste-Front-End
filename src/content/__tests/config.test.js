"use strict"

import Config from "../configuracoes/config"

it('map() should return []', () => {
    expect(Config()).toMatchSnapshot()
})
