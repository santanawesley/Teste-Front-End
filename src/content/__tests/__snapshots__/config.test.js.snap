// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`map() should return [] 1`] = `
<div
  className="configuracoes"
>
  <h1>
     CONFIGURAÇÕES 
  </h1>
  <p>
     PÁGINA EM CONSTRUÇÃO 
  </p>
</div>
`;
